src/cookbook/design/themes.md

```diff
@@ -89,44 +135,52 @@ Theme(
 );
 ```
 
-### Extending the parent theme
+### Extend the parent theme
 
-Rather than overriding everything, it often makes sense to extend the parent
-theme. You can handle this by using the [`copyWith()`][] method.
+Instead of overriding everything, consider extending the parent theme.
+To extend a theme, use the [`copyWith()`][] method.
 
 <?code-excerpt "lib/theme.dart (ThemeCopyWith)"?>
 ```dart
 Theme(
-  // Find and extend the parent theme using `copyWith`. See the next
-  // section for more info on `Theme.of`.
-  data: Theme.of(context).copyWith(splashColor: Colors.yellow),
+  // Find and extend the parent theme using `copyWith`.
+  // To learn more, check out the next
+  // section on `Theme.of`.
+  data: Theme.of(context).copyWith(
+      // TRY THIS: Change the seedColor to "Colors.red" or
+      //           "Colors.blue".
+      colorScheme: ColorScheme.fromSeed(seedColor: Colors.pink)),
   child: const FloatingActionButton(
     onPressed: null,
     child: Icon(Icons.add),
   ),
 );
 ```
 
-## Using a Theme
+## Apply a Theme
 
-Now that you've defined a theme, use it within the widgets' `build()`
-methods by using the `Theme.of(context)` method.
+To apply your new theme, add the `Theme.of(context)` method within the
+widgets' `build()` methods.
 
-The `Theme.of(context)` method looks up the widget tree and returns
+The `Theme.of(context)` method looks up the widget tree and applies
```

Comment:
nit: "retrieves" rather than "applies." Technically, `Theme.of` just gives you a reference to the nearest ancestor Theme widget's ThemeData, but doesn't do anything else with it.